### DISCUSSION TITLE
DataTable

### DISCUSSION THREAD

**Discussion by yash-isr** [primefaces/community#3286] [PrimeFaces] [answered]
Hi,

I'm using PrimeFaces DataTable with the default filter functionality. When no data matches the filter criteria, the default behavior shows a text message like "No records found."

I want to customize this behavior to display an image instead of the text message when no data is available.

Is there a way to achieve this in PrimeFaces? If possible, please share an example or guide me on how to implement it.

Thank you!

**Comment by tandraschko** [comment, marked as answer]
You can use the `f:facet name="emptyMessage"` for it; its missing in the docs, but i will add it now

**Reply by tandraschko** [reply]
https://primefaces.github.io/primefaces/15_0_0/#/components/datatable?id=customizing-emptymessage

**Reply by yash-isr** [reply]
Thank you
